src/main/scala/piconot/external/Empty.bot

```diff
@@ -0,0 +1,7 @@
+go left until blocked on left 
+then go up until blocked on up and left 
+thenLoop go down until blocked on down 
+then go right 
+then go up until blocked on up 
+then go right 
+then go back
```

Comment:
This looks way cleaner than our implementation! I think I may be a bit biased as I've already become so accustomed to coding and the state-rule system for Picobot, but there's something about replacing everything with natural language that makes me a bit uneasy. I definitely think that someone who has no exposure to computer science would have a drastically different opinion though.